pilot/pkg/bootstrap/certcontroller.go

```diff
@@ -70,14 +70,13 @@ func (s *Server) initCertController(args *PilotArgs) error {
 			// Chiron will generate the key and certificate and save them in a secret
 			secretNames = append(secretNames, c.GetSecretName())
 			dnsNames = append(dnsNames, name)
-			namespaces = append(namespaces, args.Namespace)
```

Comment:
Basically we had a bunch of code to support watching multiple namespaces. But passed it a list of the same namespace repeated over and over